<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>654</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_modes">
       <attribute name="title">
        <string>Modes</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>40</y>
          <width>77</width>
          <height>286</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btn12H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Reset 12H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn13H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Run 13H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnZero">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Zero</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn15H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Run 15H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn11H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>File 11H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn05H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Query 05H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn18H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Restart 18H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn14H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Teach 14H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn10H">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Idle 10H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn30H">
           <property name="text">
            <string>Stop 30H</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="lblMode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>330</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Mode</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnJP">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>40</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Joint Print</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnResetRead1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>70</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset Read1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnResetRead2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset Read2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnDownload">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCalcOffset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>130</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calc OFFSET</string>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_offsets">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>160</y>
          <width>191</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnPrepare">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>510</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>TextEdit -&gt; File</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnReadJP">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>420</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Read_JP</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnRead_Offsets">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>450</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Read_Offsets</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCalc_Size">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>480</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Calc_Size</string>
        </property>
       </widget>
       <widget class="QLabel" name="lblSize">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>485</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>380</y>
          <width>179</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="spinSpeed">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSpeed">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSpeed">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_axis">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <attribute name="title">
        <string>Axis</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ1minus">
           <property name="text">
            <string>J1-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ1">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-175.000000000000000</double>
           </property>
           <property name="maximum">
            <double>175.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J1min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J1max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ1plus">
           <property name="text">
            <string>J1+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ2minus">
           <property name="text">
            <string>J2-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ2">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-130.000000000000000</double>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J2min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J2max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ2plus">
           <property name="text">
            <string>J2+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ3minus">
           <property name="text">
            <string>J3-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ3">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-120.000000000000000</double>
           </property>
           <property name="maximum">
            <double>50.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J3min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J3max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ3plus">
           <property name="text">
            <string>J3+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>240</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ4minus">
           <property name="text">
            <string>J4-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ4">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-175.000000000000000</double>
           </property>
           <property name="maximum">
            <double>175.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J4min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J4max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ4plus">
           <property name="text">
            <string>J4+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_6">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>320</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ5minus">
           <property name="text">
            <string>J5-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ5">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-120.000000000000000</double>
           </property>
           <property name="maximum">
            <double>120.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J5min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J5max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ5plus">
           <property name="text">
            <string>J5+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>400</y>
          <width>331</width>
          <height>82</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="1">
          <widget class="QPushButton" name="btnJ6minus">
           <property name="text">
            <string>J6-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="SpinJ6">
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="minimum">
            <double>-360.000000000000000</double>
           </property>
           <property name="maximum">
            <double>360.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="J6min"/>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="J6max"/>
         </item>
         <item row="1" column="0" colspan="5">
          <widget class="QSlider" name="SliderJ6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="btnJ6plus">
           <property name="text">
            <string>J6+</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_port">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>201</width>
          <height>121</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>BaudRate</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbxBaudRate">
           <property name="currentText">
            <string>300</string>
           </property>
           <item>
            <property name="text">
             <string>300</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>9600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>19200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>38400</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>56800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>115200</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbxPorts"/>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btnSave">
           <property name="text">
            <string>SAVE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btnOpenClose">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>OPEN</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxReconnect">
           <property name="text">
            <string>Auto-reconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="btnINI">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>250</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="EditINI">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>320</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="EditCFG">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>290</y>
          <width>320</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btnCFG">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>310</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>INI-file:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>CFG-file:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear_2"/>
   <addaction name="actionRxD"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear_2">
   <property name="icon">
    <iconset>
     <normalon>C:/Users/Ruuz/.designer/backup/1.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionRxD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>RxD</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
